GameDesign/Assets/Scripts/TaserMonoBehaviour.cs

```diff
@@ -0,0 +1,24 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class TaserMonoBehaviour : MonoBehaviour
+{
+    // This class is only required to change the charge of the taser
+    [SerializeField] public TaserManager taserManager = TaserManager.Instance;
+
+    void Start()
+    {
+        InvokeRepeating("chargeBattery", 1.0f, 1.0f);
+    }
+
+    private void chargeBattery()
```

Comment:
This is the method where the Start and Update behaviour of the manager should go, since this will be in charge of modifying the state in the singleton (based on time in this case)

Reply:
I removed them :)